CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ For experimental package changes, see the [experimental CHANGELOG](experimental/
 
 ### :boom: Breaking Change
 
+* feat(otlp-exporter-base): added support for sending gzip compressed content in the OTLP exporter for browser [#4493](https://github.com/open-telemetry/opentelemetry-js/pull/4493)
+  * The function signature for `sendWithXhr` has changed. If you are using `otlp-exporter-base`, please update `otlp-proto-exporter-base` as well.
+
```

Comment:
I think it's okay to leave out the last sentence. People should not be consuming the base packages directly, see:
https://github.com/open-telemetry/opentelemetry-js/blob/fec6286127129ec0082e5dd080242501f15ea0af/experimental/packages/otlp-exporter-base/README.md?plain=1#L6
 
```suggestion
* feat(otlp-exporter-base)!: added support for sending gzip compressed content in the OTLP exporter for browser [#4493](https://github.com/open-telemetry/opentelemetry-js/pull/4493)
  * The function signature for `sendWithXhr` has changed.
```

Reply:
@pichlermarc made the change.